trk/src/api/CommentsApi.js

```diff
@@ -0,0 +1,20 @@
+import firebase from "@react-native-firebase/app";
+import "@react-native-firebase/firestore";
+
+
+function CommentsApi() {
+
+  const ref = firebase.firestore().collection("comments");
+  console.log('[DATABASE] CommentsApi called');
+
+  // get comments by some field
+  function getCommentsBySomeField(field, value) {
+    return ref.where(field, '==', value).get();
+  };
+
+  return {
+    getCommentsBySomeField,
+  };
+}
+
+export default CommentsApi;
```

Comment:
function which fetches comments document from the comments collection based on some field and the value this field should have